app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -37,10 +37,9 @@ def environment_url
     end
 
     def embed_js
-      if BoltConfiguration.environments[environment].zero?
-        'https://connect.bolt.com/embed.js'
-      else
-        'https://connect-sandbox.bolt.com/embed.js'
+      case BoltConfiguration.environments[environment]
```

Comment:
If we don't allow the `nil` value here, why we didn't change the column default value to 1 and preventing the column to be nil on the DB?

Reply:
Modified here [3ae75cd](https://github.com/nebulab/solidus_bolt/commit/3ae75cd88ac96ef2cdfef8f55769d8a451051f8b)